TAGradingServer/account/admin-latedays-exceptions.php

```diff
@@ -0,0 +1,752 @@
+<?php
+
+//Author: Peter Bailie, Systems Programmer, RPI Computer Science, July 2016
+
+/* MAIN ===================================================================== */
+
+include "../header.php";
+
+check_administrator();
```

Comment:
This function causes the script to die if the administrator check fails. The if statement is extraneous and the else branch will never be executed (as the script would die)